src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -295,6 +303,22 @@ trait ClusterComponent extends LeoComponent {
         .result map { recs => unmarshalFullCluster(recs)}
     }
 
+    def existsClustersWithWelderEnabled(project: GoogleProject): DBIO[Boolean] = {
+      clusterQuery
+        .filter { _.googleProject === project.value }
```

Comment:
By the way, you probably want to filter on _active_ clusters here. We don't care if welder is enabled/disabled on deleted clusters.